ui/app/pages/home/index.scss

```diff
@@ -36,4 +36,69 @@
   &__tab {
     flex-grow: 1;
   }
+
+  &__connect-status-text {
+    display: flex;
+    flex-direction: column;
+    @extend %content-text;
+    padding-left: 24px;
+    padding-right: 24px;
+
+    div {
+      &:last-child {
+        margin-top: 26px;
+      }
+    }
+  }
+
+  &__connected-status-popover {
+    width: 329px;
+    height: 295px;
+    margin-top: -12px;
+
+    .popover-header {
+      padding-bottom: 4px;
+    }
+
+    .popover-content {
+      overflow-y: auto;
+    }
+
+    .popover-arrow {
+      top: -6px;
+      left: 24px;
+    }
+
+    .popover-footer {
+      justify-content: flex-end;
+
+      & :only-child {
+        margin: 0;
+      }
+
+      button {
+        height: 39px;
+        width: 133px;
+        border-radius: 39px;
```

Comment:
Nit: It looks a bit strange to have just this button rounded, while the rest are still rectangular. I know this matches the design, but there's a separate ticket to update the button designs across the entire app, and that would seem like the better time to make the buttons rounded rather than doing it in just this one place.